src/server/frontend_xwayland/xwayland_surface.cpp

```diff
@@ -860,93 +849,23 @@ void mf::XWaylandSurface::is_transient_for(xcb_window_t transient_for)
 {
     std::shared_ptr<scene::Surface> parent_scene_surface; // May remain nullptr
```

Comment:
`parent_scene_surface` is now unused (not sure why the compiler doesn't complain)